.github/workflows/bundle-stats.yml

```
@@ -0,0 +1,57 @@
+    outputs:
+      stats:
+        description: "Bundle stats comparison"
+        value: ${{ jobs.bundle-stats.outputs.stats }}
+
+jobs:
+  bundle-stats:
+    runs-on: ubuntu-latest
+    outputs:
+      stats: ${{ steps.compare.outputs.stats }}
+    steps:
+      - uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)